README.md

```diff
@@ -237,7 +234,9 @@ test('a test that creates asynchronous activity', t => {
 The Node.js test runner can be invoked from the command line:
 
 ```bash
-node-core-test
+node-core-test --test
```

Comment:
I think this is confusing and unexpected of a node module. I understand why you added it, so that the entry points are designed the same way, however I don't see how `node-core-test` without the `--test` flag would be useful 🤔

Reply:
Ah I see 🤔  It does make sense to keep it this way in order to mirror core, however with this module in separation I'd expect the api to be `node-core-test --only` for `--test-only` behavior. What do you think?

Reply:
Well there are (currently) three possibilities: `--test`, `--test --test-only`, and `--test-only`. Having a binary that mimics core helps with the tests, so we should keep one that does that. That being said, I agree we should consider improving the API where we can and where it makes sense, but also I don't think it needs to happen in this PR (once this land, I can start the procedure to integrate this repo in the nodejs org and ask for taking over the `test` npm package, the API redesign can happen in parallel).

Reply:
Ok I follow now. Thanks for taking the time, let's go ahead